rockcraft/project.py

```diff
@@ -101,6 +101,29 @@ def _validate_build_base(cls, build_base, values):
             build_base = values.get("base")
         return build_base
 
+    @pydantic.validator("parts", pre=True)
+    @classmethod
+    def _add_pebble_part(cls, parts):
+        """Force pebble part to ensure Pebble is always installed."""
+        # TODO: by adding this part at this stage, if there is an error
+        # with it, a custom ProjectValidationError exception will be thrown,
+        # with the message "Bad rockcraft.yaml content", which is misleading
+        # since the user hasn't specified such part.
+        # Is there another logical place where this insertion could happen?
+        pebble_part_spec = {
+            "pebble": {
+                "plugin": "go",
+                "source": "https://github.com/canonical/pebble.git",
+                "build-environment": [
+                    {"CGO_ENABLED": "0"},
+                ],
+                "build-snaps": ["go"],
```

Comment:
lock to a channel for this not to unexpectedly break

Reply:
good idea. done